infra/terraform/modules/data_access/teams/teams.py

```diff
@@ -62,6 +62,15 @@ def create_team_bucket(event, context):
         ACL="private",
         CreateBucketConfiguration={"LocationConstraint": region},
     )
+    client.put_bucket_logging(
+        Bucket=name,
+        BucketLoggingStatus={
+            'LoggingEnabled': {
+                'TargetBucket': os.environ.get('LOGS_BUCKET_NAME'),
```

Comment:
What happens if `LOGS_BUCKET_NAME` is `None` (the default return value of `get` if the env var is not set? Should this be `os.environ['LOGS_BUCKET_NAME']` instead, to raise a `KeyError` exception if it is not set?